### ISSUE TITLE
On Windows?

### ISSUE THREAD

**Issue by natny** [YerevaNN/mimic3-benchmarks#15]
Hrant et al., thanks so much for writing this amazing  gateway paper. I am very new to machine learning. Has anyone tried out executing this code on windows yet? Iv'e had multiple issues (from unzipping on windows, theano on windows etc) , but most were fixable with a powerful search engine and common sense. However, I am not sure which of those is failing me now: 
I finished training the 2 layer LSTM for phenotyping model on my personal instance of MIMIC III  for 30 epochs and have their various states stored in the state folder. now do I rename one of them to best_model.state and drop it in the phenotyping folder? I did so but got an EOFError when I tried to cPickle.load them. Kindly help!

**Comment by natny** [comment]
- [ ]

**Comment by Hrant-Khachatrian** [comment]
Thanks natny!

We never tested this on Windows, so I can't give a good feedback on that.

Regarding the name of the state file, no need to rename it. Just pass the path to your model in the command line

**Comment by natny** [comment]
Thanks very much for the quick response Hrant. Oh yes, that too, but I was wondering if my state files are somehow corrupt. Is there any way I could get my hands on one of your state files?

**Comment by natny** [comment]
For anyone else crazy enough to implement on windows, the answer's on this page: https://stackoverflow.com/questions/8527241/python-pickle-fix-r-characters-before-loading. These errors are because windows pickles and unpickles (serialization) differently and the way to make the code more portable is to use 'wb' and 'rb' for state files.

**Comment by Nearby36** [comment]
I met this problem as @natny described when I ran the code using GitBash in Windows 7.
As the error mentioned that 'best_model.state' was missing, I chose the state file with best performance (from the paper) and renamed it as 'best_model.state'.  However, an EOFError came out. The error is related to pickles.
Plus, in windows, files named like 'decomp_ts0.8.input_str_previous.n1e5.start_time_zero.normalizer' is different from Linux. when importing these files, ":" should be replaced by '_'.
Thanks!
